### ISSUE TITLE
main_follow

### ISSUE THREAD

**Issue by JDCodeIt** [sapier/mobf_core#1]
I noticed that an undeclared global accel_to_set gets a value of a zero vector, but then it is not used in this function - line 595 if main_follow.lua.  I'm not sure that is what you intended. Was the idea that if the current position is OK, then don't accelerate?  Maybe accel_to_set is supposed to be "accel"

```lua
	elseif mgen_follow.next_block_ok(entity,pos,{x=0,y=0,z=0}) then
		accel_to_set = {x=0,y=0,z=0}
		dbg_mobf.fmovement_lvl3("MOBF:   setting acceleration to: " .. printpos(accel));
		entity.object:setacceleration(accel)
		return true
	else
```

**Comment by sapier** [comment]
you're right, this was a bug introduced by refactoring, fixed in current version
